### PR TITLE
[ACTV-10128] Add datamanager login

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by paul-ciobanu — APPROVED** [review]

File: database/createDatabases.ps1 [hunk 1]
```diff
@@ -150,13 +155,19 @@ if($singleDatabase -eq "N"){
     "
 }
 else {
+$dmCommand = ""
+if ($withDatamanager -eq $true){
+    $dmCommand = "CREATE LOGIN $datamanager_login_user WITH PASSWORD=N'{{pwd}}'"
```
[the remark this answers is not in the document]
Comment (irinel-birsan-uipath) [reply database/createDatabases.ps1:160]: Yes, that's right

[the remark this answers is not in the document]
Comment (bogdancondurache) [reply database/createDatabases.ps1:160]: no, that's bad, very bad :))

[the remark this answers is not in the document]
Comment (bogdancondurache) [reply database/createDatabases.ps1:160]: https://lmgtfy.app/?q=why+is+it+bad+to+use+the+same+password

[the remark this answers is not in the document]
Comment (irinel-birsan-uipath) [reply database/createDatabases.ps1:160]: I don't we can change this easily. The installation script receives only one password.

**Review by tephe — APPROVED** [review]

**Review by vnaren23 — APPROVED** [review]
LGTM

**Review by Mdirfan6 — APPROVED** [review]
lgtm

**Pull request merged**